airflow/values.yaml

```diff
@@ -0,0 +1,167 @@
+# Default values for airflow-helm
+
+replicas:
+  web: 1
+  worker: 1
+  flower: 1
+  scheduler: 1
+
+labels:
+  node_selector_key: openstack-control-plane
+  node_selector_value: enabled
+
+images:
+  airflow: docker.io/eanylin/airflow-helm:1.8.1-1.6.2-0.2
+  dep_check: quay.io/stackanetes/kubernetes-entrypoint:v0.1.1
+  db_init_postgresql: docker.io/postgres:9.5
+  db_sync: docker.io/eanylin/airflow-helm:1.8.1-1.6.2-0.2
+  pull_policy: "IfNotPresent"
+
+network:
+  flower:
+    port: 5555
+    node_port: 32081
+    enable_node_port: true
+  web:
+    port: 8080
+    node_port: 32080
+    enable_node_port: true
+
+volumes:
+  path: /home/ubuntu/workbench/dags
+
+scheduler_args: '5'
+
+upgrades:
+  revision_history: 3
+  pod_replacement_strategy: RollingUpdate
+  rolling_update:
+    max_unavailable: 1
+    max_surge: 3
+
+dependencies:
+  db_init_postgresql:
+    jobs:
+    service:
+    - postgresql
+  db_sync:
+    jobs:
+    - airflow-db-init-postgresql
+    service:
+    - postgresql
+
+database:
+  db_name: airflow
+  db_user: airflow
+  db_pass: airflow
+  db_fqdn: postgresql.airflow
+  root_db_user: postgres
+
+conf:
+  core:
+    airflow_home: /usr/local/airflow
+    dags_folder: /usr/local/airflow/dags
+    base_log_folder: /usr/local/airflow/logs
+    remote_base_log_folder:
+    remote_log_conn_id:
+    encrypt_s3_logs: False
+    executor: CeleryExecutor
+    sql_alchemy_conn: postgresql+psycopg2://airflow:airflow@postgresql.airflow/airflow
+    sql_alchemy_pool_size: 5
+    sql_alchemy_pool_recycle: 3600
+    parallelism: 32
+    dag_concurrency: 16
+    dags_are_paused_at_creation: False
+    max_active_runs_per_dag: 16
+    load_examples: False
+    plugins_folder: /usr/local/airflow/plugins
+    fernet_key: $FERNET_KEY
+    donot_pickle: False
+    dagbag_import_timeout: 30
+  webserver:
+    base_url: http://localhost:8080
+    web_server_host: 0.0.0.0
+    web_server_port: 8080
+    web_server_worker_timeout: 120
+    secret_key: temporary_key
+    workers: 4
+    worker_class: sync
+    expose_config: true
+    authenticate: False
+    filter_by_owner: False
+  email:
+    email_backend: airflow.utils.send_email_smtp
+  smtp:
+    smtp_host: localhost
+    smtp_starttls: True
+    smtp_ssl: False
+    smtp_user: airflow
+    smtp_port: 25
+    smtp_password: airflow
+    smtp_mail_from: airflow@airflow.local
+  celery:
+    celery_app_name: airflow.executors.celery_executor
+    celeryd_concurrency: 16
+    worker_log_server_port: 8793
+    broker_url: amqp://airflow:airflow@rabbitmq.airflow:5672/airflow
```

Comment:
If you copy the routines to build endpoints from helm-toolkit, these need not be defined, and amqp can be defined within endpoints.

see 

https://github.com/openstack/openstack-helm/blob/master/helm-toolkit/templates/_endpoints.tpl#L43-L76

and

https://github.com/openstack/openstack-helm/blob/master/glance/values.yaml#L306-L319

and

https://github.com/openstack/openstack-helm/blob/master/glance/templates/configmap-etc.yaml#L46-L48

Same thing applies for any postgre URLs.